src/app.js

```diff
@@ -1 +1,47 @@
+/* eslint-disable no-console */
 'use strict';
+
+const paramsFunction = process.argv.slice(2, 3)[0];
+const paramsValues = process.argv.slice(3).reduce((acc, total) => {
+  if (!isNaN(+total)) {
+    acc.push(+total);
+  }
+
+  return acc;
+}, []);
+
+if (paramsFunction === 'add') {
```

Comment:
It would be great to use switch...case instead of else...if as you have a limited amount of params